docusaurus/docs/quickstarts/zac/installation.md

```diff
@@ -33,32 +31,31 @@ you can perform the following steps.
     ```bash
     git clone https://github.com/openziti/ziti-console.git "${ZITI_HOME}/ziti-console"
     ```
-   
-2. Install node and npm and get the server ready:
+
+1. Install Node modules:
 
     ```bash
     cd "${ZITI_HOME}/ziti-console"
-    sudo apt install npm nodejs -y
     npm install
     ````
-   
-3. Use the ziti-controller certificates for the Ziti Console:
+
+1. Use the ziti-controller certificates for the Ziti Console:

```

Comment:
this could probably be enhanced to just talk about how the certs need to be configured for ZAC and then state that you can just use the same certs from the controller, but they browser won't automatically trust them. i think that's a whole separate page that might be useful though, and not one that I would stuff all in here. Mabe just a "see here how to configure ZAC for TLS" or something like taht?

Reply:
Sounds like the beginnings of a ZAC deployment guide for the new Ops section.